src/util/jira-utils.ts

```diff
@@ -43,6 +43,11 @@ export const getJiraAuthor = (...authors: (Author | undefined)[]): JiraAuthor =>
 	};
 };
 
+export const limitCommitMessage = (message: string): string => {
+	const MAX_COMMIT_MESSAGE_LENGTH = 1024;
```

Comment:
I'd probably add length as an input to the function with a default of 1024: `(message = "", length = 1024):string`